src/ngx_http_lua_module.c

```diff
@@ -748,6 +755,11 @@ ngx_http_lua_init(ngx_conf_t *cf)
         dd("Lua VM initialized!");
     }
 
+    if (lmcf->requires_intercept_log) {
+        cf->cycle->log_intercept_handler = (ngx_log_intercept_pt)
```

Comment:
BTW, we should protect such code with our own C macros so that this nginx module can still be compiled with the standard nginx core. And when working with the stock nginx cores, we should emit an error when the user configures this config directive that requires our nginx core patch.